blazor/datagrid/pdf-export.md

```diff
@@ -1059,6 +1059,107 @@ To apply theme in exported PDF, define the **theme** in export properties.
 
 > By default, material theme is applied to exported PDF document.
 
+### How to customize header in exported document
+
+PDF export provided an option to rotate the header text or cell contents in the exported document. This can be achieved by using the BeginCellLayout property of the [PdfExportProperties](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.PdfExportProperties.html) class.
```

Comment:
Now mentioned the event name with in back tick like this `BeginCellLayout`. Later we will provide api link

Reply:
Provided back tick line to **BeginCellLayout**